data/json/recipes/practice/skills.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "id": "prac_computer_beg",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (beginner)",
+    "description": "Practice the basics of using a computer",
+    "skill_used": "computer",
+    "time": "30 m",
```

Comment:
```suggestion
    "time": "1 h",
```

Reply:
Going from level 0 to 1 takes about 15 minutes, so unless I add an "end practice on level up" feature it's going to either waste 45 minutes or require the player to hit the interrupt button very quickly.

Reply:
Fair, we could keep the lower levels shorter if you can't get that feature in.